elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -163,13 +202,44 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
         }
     }
 
+    @Override
+    public void registerOperations(ManagementResourceRegistration resourceRegistration) {
+        super.registerOperations(resourceRegistration);
+        ResourceDescriptionResolver resolver = ElytronExtension.getResourceDescriptionResolver(ElytronDescriptionConstants.FILESYSTEM_REALM);
+        if (isServerOrHostController(resourceRegistration)) { // server-only operations
+            ChangeKeyPairHandler.register(resourceRegistration, resolver);
+        }
+    }
+
+    static class ChangeKeyPairHandler extends ElytronRuntimeOnlyHandler {
```

Comment:
Just a minor comment but maybe this could be called `UpdateKeyPairHandler` instead (just to match the operation name).